rpa-manager/modules/ROOT/pages/processautomation-prepare-deployment-credential.adoc

```diff
@@ -0,0 +1,179 @@
+
+== Create a Credential
+
+Create credentials with which a bot can log in to an external application during process runs.
+
+To create a credential:
+
+. Open the *Credential Pool* view of the *Process Automation* module.
+. Click *Create*.
+. [[form-create-credential]] Complete the *Create a New Credential* form:
+* [[form-create-credential-useraffiliation]] *User affiliation*
++
+Click to choose users that may view, edit, and use the credential. Add yourself so that you can also view, edit, and use the credential.
+* *User group affiliation*
++
+Click to choose user groups whose users may view, edit, and use the credential.
+* *Phase affiliation*
++
+Phases in which the credential can be deployed. When you create or change the credential, your privileges determine which phases the user can select:
++
+** *Test*
++
+The credential can be deployed in the test phase.
++
+** *Production*
++
+The credential can be deployed in the production phase.
+
+* *Type*
++
+Type of the global credential. The type of credential determines where you can use it. The original value defined in RPA Builder is typed. Select one of the following types:
++
+** *AWS Credentials*
++
+Use this credential for accessing Amazon Web Services. Fill in the client ID, client secret, and region. Using a session token is optional.
+** *OAuth Credentials*
++
+Use this credential for accessing the Microsoft Outlook email application via Microsoft Azure.
++
+*** *OAuth Host*
++
+The host address used to get the credentials. The default address for Outlook services is `https://login.microsoftonline.com/common/oauth2/v2.0/`. This address might change if the customer uses a self hosted service.
+*** *Client ID*
++
+To authenticate with the OAuth Host, an Azure application must be defined in the customer's space. This Azure application has an ID, which is unique in the entire OAuth Host space, and that allows users to explicitly grant or revoke access for this Azure application to their accounts.
++
+For more information about how to create this Azure application for Outlook, see https://learn.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app[Quickstart: Register an application with the Microsoft identity platform^].
+*** *Client Secret*
++
+This secret enables RPA Manager to prove to the OAuth Host that it received the permission to access user accounts on behalf of the registered Azure application. Without it, the authentication is not possible.
+*** *Redirect URI*
++
+The URI to which the OAuth Host redirects after the user completes the authentication attempt. This redirect URI must be registered with the Azure application. The OAuth Host allows the authentication only if the entered redirect URI matches one of the registered URIs.
+*** *Scopes*
++
+The scopes define which permission the user grants to the Azure application. Grant the following permissions:
```

Comment:
```suggestion
The scopes define which of the following permissions the user grants to the Azure application:
```

Reply:
OK. It's just that the first sentence has "user grants" and the second sentence is imperative (implied "you grant"). It's unclear (to me) how they relate, which is why I was suggesting combining the two.

Reply:
Waiting for developer feedback on replacement suggestion:
Scopes are permission sets for an Azure application. Grant the following scopes and permissions to the Azure application you created for the authentication: